pkg/cluster/gatherlogs.go

```diff
@@ -26,7 +30,9 @@ func (m *manager) gatherFailureLogs(ctx context.Context) {
 		{f: m.logNodes, isJSON: true},
 		{f: m.logClusterOperators, isJSON: true},
 		{f: m.logIngressControllers, isJSON: true},
+		{f: m.logPodLogs, isJSON: false},
 		{f: d.LogVMSerialConsole, isJSON: false},
+		{f: m.logPodLogs, isJSON: false},
```

Comment:
Do we need to check the logs twice? Or is there another reason this is here twice?

Reply:
Thanks for taking a look, I was testing this while figuring out this fix https://github.com/Azure/ARO-RP/commit/29f922900d79bf73b711e68ff53a0ca12cdb348e. I'm going to add a temporary revert of this commit so we have a recently-failed e2e run with pod log collection.

The duplicate is from a rebase gone wrong I'm sure, will fix.

Reply:
Fixed duplicate call